c7n/filters/metrics.py

```diff
@@ -150,7 +150,7 @@ def get_metric_window(self):
         schedule defined here:
 
         https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_concepts.html#Metric
-        """ # noqa: E501
+        """  # noqa: E501
```

Comment:
ruff isn't good at dealing with whitespace issues I'm noticing.

Reply:
ruff sort of assumes use of a formatter (aka black) on a codebase, re doing that #8281